source/solvers/cahn_hilliard_scratch_data.cc

```diff
@@ -66,6 +66,78 @@ CahnHilliardScratchData<dim>::allocate()
     n_q_points, std::vector<Tensor<2, dim>>(n_dofs));
   this->laplacian_phi_potential =
     std::vector<std::vector<double>>(n_q_points, std::vector<double>(n_dofs));
+
+  // Velocity values
+  this->velocity_values = std::vector<Tensor<1, dim>>(this->n_q_points);
+  this->previous_velocity_values = std::vector<std::vector<Tensor<1, dim>>>(
+    maximum_number_of_previous_solutions(),
+    std::vector<Tensor<1, dim>>(this->n_q_points));
+  this->velocity_gradient_values =
+    std::vector<Tensor<2, dim>>(this->n_q_points);
+}
+template <int dim>
+void
+CahnHilliardScratchData<dim>::calculate_physical_properties()
```

Comment:
This should be reviewed (CHNS implementation).

Reply:
This piece of code was written in advance but serve no purpose here. It should be moved in the next PR

Reply:
edit  : it was replaced by an empty function to implement it properly in the next PR which will cover the Navier-Stokes coupling